sigmf-spec.md

```diff
@@ -79,6 +79,10 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 JSON keywords are used as defined in [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
 
+Fields defined as "human-readable", a "string", or simply as "text" shall be
+treated as plaintext ASCII where whitespace is respected, unless otherwise
```

Comment:
Referring to “ASCII” is not just over-constraining but incorrect here. The contents of a JSON document are not bytes to be interpreted via an encoding — they are characters.

I would suggest “…shall be treated as plain text where all whitespace is significant, …”